{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Exception"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/swiftsoup\/exception"],"traits":[{"interfaceLanguage":"swift"}]}],"relationshipsSections":[{"identifiers":["doc:\/\/com.jaemyeong.SDSMealAPI\/s5ErrorP","doc:\/\/com.jaemyeong.SDSMealAPI\/s8SendableP"],"kind":"relationships","title":"Conforms To","type":"conformsTo"}],"identifier":{"url":"doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup\/Exception","interfaceLanguage":"swift"},"topicSections":[{"title":"Enumeration Cases","identifiers":["doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup\/Exception\/Error(type:Message:)"]},{"title":"Default Implementations","identifiers":["doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup\/Exception\/Error-Implementations"],"generated":true}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Exception"}],"title":"Exception","roleHeading":"Enumeration","role":"symbol","symbolKind":"enum","externalID":"s:9SwiftSoup9ExceptionO","modules":[{"name":"SwiftSoup"}],"navigatorTitle":[{"kind":"identifier","text":"Exception"}]},"hierarchy":{"paths":[["doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup"]]},"references":{"doc://com.jaemyeong.SDSMealAPI/s8SendableP":{"type":"unresolvable","title":"Swift.Sendable","identifier":"doc:\/\/com.jaemyeong.SDSMealAPI\/s8SendableP"},"doc://com.jaemyeong.SDSMealAPI/documentation/SwiftSoup/Exception/Error(type:Message:)":{"role":"symbol","title":"Exception.Error(type:Message:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"Error"},{"kind":"text","text":"("},{"kind":"externalParam","text":"type"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"ExceptionType","preciseIdentifier":"s:9SwiftSoup13ExceptionTypeO"},{"kind":"text","text":", "},{"kind":"externalParam","text":"Message"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"abstract":[],"identifier":"doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup\/Exception\/Error(type:Message:)","kind":"symbol","type":"topic","url":"\/documentation\/swiftsoup\/exception\/error(type:message:)"},"doc://com.jaemyeong.SDSMealAPI/s5ErrorP":{"type":"unresolvable","title":"Swift.Error","identifier":"doc:\/\/com.jaemyeong.SDSMealAPI\/s5ErrorP"},"doc://com.jaemyeong.SDSMealAPI/documentation/SwiftSoup/Exception":{"role":"symbol","title":"Exception","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"Exception"}],"abstract":[],"identifier":"doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup\/Exception","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"Exception"}],"url":"\/documentation\/swiftsoup\/exception"},"doc://com.jaemyeong.SDSMealAPI/documentation/SwiftSoup":{"role":"collection","title":"SwiftSoup","abstract":[],"identifier":"doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup","kind":"symbol","type":"topic","url":"\/documentation\/swiftsoup"},"doc://com.jaemyeong.SDSMealAPI/documentation/SwiftSoup/Exception/Error-Implementations":{"role":"collectionGroup","title":"Error Implementations","abstract":[],"identifier":"doc:\/\/com.jaemyeong.SDSMealAPI\/documentation\/SwiftSoup\/Exception\/Error-Implementations","kind":"article","type":"topic","url":"\/documentation\/swiftsoup\/exception\/error-implementations"}}}